data/json/proficiencies/chemistry.json

```diff
@@ -8,5 +28,65 @@
     "default_time_multiplier": 2,
     "default_fail_multiplier": 2.5,
     "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_mutagen",
+    "name": { "str": "Mutagen Production" },
+    "description": "Knowledge on the production of general mutagens.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "12 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_serum",
+    "name": { "str": "Serum Production" },
+    "description": "Knowledge on the synthesization of the key to the perfect self.",
+    "can_learn": true,
+    "default_time_multiplier": 2.5,
+    "default_fail_multiplier": 5,
+    "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_fuel",
```

Comment:
This should be a recipe, not a proficiency. The proficiency should be the techniques used to make fuels, not the general concept of 'fuel' itself.